OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -244,3 +279,84 @@ Function& LiveFunctionsDataView::GetFunction(unsigned int a_Row) const {
   CHECK(m_Functions[m_Indices[a_Row]]);
   return *m_Functions[m_Indices[a_Row]];
 }
+
+void LiveFunctionsDataView::JumpToBox(const TextBox* box) const {
+  if (box) {
+    GCurrentTimeGraph->Zoom(box);
+  }
+}
+
+std::pair<TextBox*, TextBox*> LiveFunctionsDataView::GetMinMax(
+    Function& function) const {
+  auto function_address = function.GetVirtualAddress();
+  TextBox *min_box = nullptr, *max_box = nullptr;
+  std::vector<std::shared_ptr<TimerChain>> chains =
+      GCurrentTimeGraph->GetAllThreadTrackTimerChains();
+  for (auto& chain : chains) {
+    for (TimerChainIterator it = chain->begin(); it != chain->end(); ++it) {
```

Comment:
There was a 
if (!chain) continue; 
here, before, right?

Are you sure you can never get a nullptr here? If yes, add a CHECK(chain), but you'd have to really make sure that you can not get a nullptr chain from GetAllThreadTrackTimerChains().

Reply:
Yes, it was there. I don't know why I delete it. I think that there isn't going to be a nullptr, but I dont lose anything having it.